Gauge.CSharp.Lib/ExecutionContext.cs

```diff
@@ -7,6 +7,7 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
```

Comment:
Dont think there is any use for Diagnostics in this file, or am I missing something?

Reply:
Fixed